.mergify.yml

```diff
@@ -24,7 +24,7 @@ pull_request_rules:
         add: ["auto merged"]
```

Comment:
@rahul-chekuri please add logic to merge (not squash) PRs with the `no-squash` label.

Reply:
done 

https://github.com/spinnaker/spinnaker/pull/7032/commits/4f8a42bfa0144ed76a12caf1ebcae3964bc5398c